test-e2e/tests/payments/subscription_test.ts

```diff
@@ -30,8 +30,6 @@ runTestSuite(jwProps, 'JW Player');
 runTestSuite(cleengProps, 'Cleeng');
 
 function runTestSuite(props: ProviderProps, providerName: string) {
-  let paidLoginContext: LoginContext;
```

Comment:
Removing this means that every test here will get a new user created. It's probably not the end of the world in regard to creating junk data, but it does mean we aren't testing any scenarios for 'existing users'.  Was the simultaneous login feature causing issues?  If so, maybe we can better just add a "Logout" to the end of each test.